routes.go

```
@@ -0,0 +1,68 @@
+	serveHTMLFile(w, "/about.html")
+}
+
+func personHandler(w io.Writer, r *webserver.Request) {
+	id := r.UrlValues["id"]
+	response := webserver.NewResponse(200, []byte("Person with id: "+id))
+	response.WriteTo(w)
+}
+
+func peopleHandler(w io.Writer, r *webserver.Request) {
+	response := webserver.NewResponse(200, []byte("People"))
+	response.WriteTo(w)
```

golangci-lint (1.62.2)

Error return value of `response.WriteTo` is not checked

(errcheck)